ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesView.java

```diff
@@ -41,7 +41,7 @@ public String render(final GetTapesResult obj) {
         }
         this.tapeList = result.getTapes();
 
-        initTable(ImmutableList.of("Bar Code", "ID", "State", "Last Modified", "Available Raw Capacity", "BucketID", "Assigned to Storage Domain" ));
+        initTable(ImmutableList.of("Bar Code", "ID", "State", "Last Modified", "Available Raw Capacity", "BucketID", "Assigned to Storage Domain", "Ejection Date", "Ejection Location", "Ejection Label"));
```

Comment:
No, most views just use defaults. I added it to make the old commands not change. If you don't call it, you're fine.